app/views/admin/dashboard/index.html.haml

```diff
@@ -42,33 +42,39 @@
         %li
           = link_to t('admin.dashboard.feature_registrations'), edit_admin_settings_path
           - if @registrations_enabled
-            %span.pull-right.positive-hint= fa_icon 'check fw'
+            %span.pull-right.positive-hint= fa_icon 'power-off fw'
```

Comment:
power-on, I guess?

Reply:
There is no power-on icon. They are the same thing

Reply:
Ah, I see. So the only difference is going to be the color? I'm afraid that it could be an accessibility issue

Reply:
Now they also say "yes"/"no"